packages/ses/src/compartment-shim.js

```diff
@@ -27,9 +27,12 @@ import { getDeferredExports } from './module-proxy.js';
 import { assert } from './error/assert.js';
 import { compartmentEvaluate } from './compartment-evaluate.js';
 import { makeSafeEvaluator } from './make-safe-evaluator.js';
+import {makeHardener} from './make-hardener.js';
```

Comment:
Running `yarn format` at the base of the repo will get you through CI. Your favorite IDE can be configured to do this automatically when you save a file. My favorite IDE can too, but I haven’t figured it out.